.tool-versions

```diff
@@ -0,0 +1,2 @@
+erlang 25.3.2
```

Comment:
Q: shouldn't it match our OTP fork?

```suggestion
erlang 25.3.2-2
```

Reply:
not sure, do we require to use our otp fork in order to build emqx plugins?

Reply:
Since EMQX is a dependency, I'd think it might be consistent to use it as well.  But it might not be required, indeed, as OTP stuff probably won't appear in the resulting BEAMs.  :thinking: